components/Button/Button.test.tsx

```diff
@@ -2,14 +2,30 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { Button } from './Button';
 
-test('should render a button with no variant defined', () => {
-  render(<Button />);
-  const defaultButton = screen.getByRole('button');
-  expect(defaultButton).toHaveClass('bg-orange-600');
-});
-
-test('should render a button with the link variant defined. It should have no background or border, only an underline on hover', () => {
-  render(<Button variant="link" />);
-  const defaultButton = screen.getByRole('button');
-  expect(defaultButton).toHaveClass('underline-offset-4');
+describe('Button Component', () => {
+  it('renders the default variant', () => {
+    render(<Button label="Default Variant" />);
+    const button = screen.getByText(/Default Variant/i);
```

Comment:
S: Could you add `data-testid` attribute to the original Button component with an appropriate `data-testid` value, and then grab the component using `.getByTestId()`?

Reply:
Tell me why `data-testid` is better here than label.

Reply:
To take a quote directly from the following article source (https://bugbug.io/blog/software-testing/data-testid-attributes/):

"The primary use of data-testid attributes is to improve the robustness and reliability of UI (User Interface) tests. By using these identifiers, tests can locate and interact with specific elements on a page without relying on CSS classes or element tags, which can change more frequently and be less unique. This approach helps to create more maintainable and stable test suites, as tests are less likely to break due to changes in the application's UI."